scripts/install/sonarr.sh

```diff
@@ -19,7 +19,39 @@
 #
 #################################################################################
 
+function _check_for_sonarr3 () {
+  if [[ -f /install/.sonarrv3.lock ]]; then 
+    v3present=true
+  fi
+
+  if dpkg -l | grep sonarr > /dev/null 2>&1 ; then 
+    v3present=true
+  fi
+
+  if [[ $v3present == "true" ]]; then
+    echo "Sonarr v3 detected. If you want to proceed installing Sonarr v2, please remove Sonarr v3 first."
+    exit 1
+  fi
+
+  if [[ -d /root/sonarrv2.bak ]]; then 
+    echo
+    echo "WARNING: Found backups before Sonarr v3 installation"
+    echo "Follow these steps post-install https://github.com/Sonarr/Sonarr/wiki/Backup-and-Restore"
+    echo "The backup is stored in /root/sonarrv2.bak/"
+    echo
+    #TODO implement restore procedure if user wants that to happen?
+    #TODO check if unit is still masked. Unmaks it if that's the case.
+  fi
+
+}
+
 function _installSonarrintro() {
+  echo "Sonarr v3 is available as a separate application (sonarrv3)"
+  . /etc/swizzin/sources/functions/ask
+  if ! ask "Would you like to continue installing Sonarr v2?"; then
```

Comment:
Does this prompt a question during setup? Ideally this should be avoided because we have to assume the user specifically chose v2 because v3 would be listed right beneath it.

Reply:
I'm assuming users won't know `sonarrv3` is even available. I never check the list, and we have no way to really inform people that new packages are rolling out. It's probablt not gonna land on the swizzin.tld main site anytime soon either, which I assume gets a lot more traffic than the docs or the github commits.

Reply:
> It's probablt not gonna land on the swizzin.tld main site

:facepalm: 

I just want to ensure minimal questions are asked during setup to make it as unattended as possible once it starts installing -- I have been doing my best to streamline and automate things where possible (see pyload) to remove user interaction for setting things up, maybe it would be possible to ask this question earlier?

Plex is a notable exception to this rule, but I generally don't install it on the first pass anymore

Reply:
> maybe it would be possible to ask this question earlier?

It should be the very first thing that gets ran, it's just that the functions are ordered weird.

Reply:
I caved in and disabled this.